doc/development-context-aware-applications/introduction.md

```diff
@@ -44,12 +44,10 @@ a context-aware application check out:
     - [How to update and query context information](v2/how-to-update-and-query-context-information.md)
     - [How to subscribe to changes on context information](v2/how-to-subscribe-to-changes-on-context-information.md)
     - [How to perform geo-location queries](v2/how-to-perform-geo-located-queries.md)
+    
+  By other hand, you can see a the full API [here](http://fiware.github.io/specifications/ngsiv2/latest/cookbook/)
 
-- Using NGSI v1: 
-    - [How to update and query context information](how-to-update-and-query-context-information.md)
-    - [How to connect to external context providers](how-to-connect-to-external-context-providers.md)
-    - [How to subscribe to changes on context information](how-to-subscribe-to-changes-on-context-information.md)
-    - [How to perform geo-location queries](how-to-perform-geo-located-queries.md)
```

Comment:
Why are you removing this content?

Reply:
again, please do not change the current structure of the document, but only add or update links